internal/sign/ocpbuild/manager.go

```diff
@@ -22,15 +22,15 @@ import (
 type manager struct {
 	client          client.Client
 	maker           Maker
-	ocpBuildsHelper buildutils.OpenShiftBuildsHelper
+	ocpBuildsHelper ocpbuildutils.OpenShiftBuildsHelper
 	authFactory     auth.RegistryAuthGetterFactory
 	registry        registry.Registry
 }
 
 func NewManager(
 	client client.Client,
 	maker Maker,
-	ocpBuildsHelper buildutils.OpenShiftBuildsHelper,
+	ocpBuildsHelper ocpbuildutils.OpenShiftBuildsHelper,
```

Comment:
Do we need the `Openshift` prefix now that we have `ocp...` in the package name?

Reply:
same answer as above

Reply:
Then how about we change `Openshift` to `OCP` to be consistent?

Reply:
fixed